### ISSUE TITLE
Cannot read property 'getBaseTypeFromSchema' of undefined

### ISSUE THREAD

**Issue by piotr121993** [rtk-incubator/rtk-query-codegen#60]
I get this error when running code generation:
```
TypeError: Cannot read property 'getBaseTypeFromSchema' of undefined
    at Array.map (<anonymous>)
    at ApiGenerator.getUnionType (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:295:70)
    at ApiGenerator.getBaseTypeFromSchema (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:321:25)
    at ApiGenerator.getTypeFromSchema (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:304:27)
    at C:\Users\Kosky\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:374:28
    at Array.map (<anonymous>)
    at ApiGenerator.getTypeFromProperties (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:368:44)
    at ApiGenerator.getBaseTypeFromSchema (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:337:25)
    at ApiGenerator.getTypeFromSchema (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\oazapfts\lib\codegen\generate.js:304:27)
```
The issue is fixed in the latest master of `oazapfts`, [Link to original issue](https://github.com/cellular/oazapfts/issues/51), but not yet released. 

Running with linked master code generates this error instead:
```
TypeError: apiGen.hasJsonContent is not a function
    at generateEndpoint (C:\UserName\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\@rtk-incubator\rtk-query-codegen-openapi\lib\generate.js:144:36)
    at C:\Users\Kosky\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\@rtk-incubator\rtk-query-codegen-openapi\lib\generate.js:129:122
    at Array.map (<anonymous>)
    at Object.generateApi (C:\Users\UserName\AppData\Local\npm-cache\_npx\3c5ed9efbe5d2091\node_modules\@rtk-incubator\rtk-query-codegen-openapi\lib\generate.js:129:93)
```

**Comment by gormonn** [comment]
I have similar problem now.
And don't understand why

**Comment by phryneas** [comment]
Could you please give the last `main` a try and verify if it works now? I would create a release based off of that then.

**Comment by piotr121993** [comment]
I tried `main` and indeed it works now.

**Comment by phryneas** [comment]
Resolved with 0.4.0
